healthScore/healthScore/urls.py

```diff
@@ -4,5 +4,6 @@
 
 urlpatterns = [
     path('index', views.test_default_values, name='index'),
-    path('addMockData', views.add_mock_data, name='mock_data')
+    path('addMockData', views.add_mock_data, name='mock_data'),
+    path('view_health_history', views.view_health_history, name='view_health_history'),
```

Comment:
path('view_health_history', views.view_health_history, name='view_health_history'),

We could have this as 
path('viewHealthHistory', views.view_health_history, name='view_health_history'),
instead. Since the first part is going to be the url part, camelCasing it might be a better option

Reply:
Fixed